packages/atomic/src/components/facets-v1/atomic-timeframe/atomic-timeframe.tsx

```diff
@@ -0,0 +1,43 @@
+import {Component, Prop} from '@stencil/core';
+import {RelativeDatePeriod, RelativeDateUnit} from '@coveo/headless';
+import {Timeframe} from './timeframe';
+
+/**
+ * The `atomic-timeframe` component defines a timeframe of an `atomic-timeframe-facet`, and therefore must be defined within an `atomic-timeframe-facet` component.
+ *
+ * A timeframe is a span of time from now to a specific time in the past.
+ */
+@Component({
+  tag: 'atomic-timeframe',
+  shadow: false,
+})
+export class AtomicTimeframe implements Timeframe {
+  /**
+   * The relative period of time.
+   */
+  @Prop() public period: RelativeDatePeriod = 'past';
```

Comment:
As a developer using this option, I feel like `now` would be ambiguous. We should probably document what it does here.

Reply:
Oh, Ill remove now from the option, forgot about this!